gdscript-godot.el

```diff
@@ -36,6 +36,14 @@
 (require 'gdscript-utils)
 
 ;;;###autoload
+(defvar gdscript-godot-debug-selected-option 1)
```

Comment:
There's a small thing I'd need you to change in your PRs: anything that's pseudo-private (shouldn't be accessed by the user) should be with two dashes `package-name--symbol-name`. It's part of the MELPA style guide. Could you change the variables and functions you added to be `gdscript-godot--*`?